cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -82,3 +82,249 @@ def test_provider_summary_topology():
         1532404
     """
     pass
+
+
+@pytest.mark.manual
```

Comment:
optional: one optional think you can do; File it self indicating `*_manual.py` so let move this marker as global marker.